### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by sushi2k**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [ ] Your contribution is written in the 2nd person (e.g. you)
- [ ] Your contribution is written in an active present form for as much as possible.
- [ ] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [ ] Your contribution has proper formatted markdown and/or code
- [ ] Any references to website have been formatted as [TEXT](URL “NAME”)
- [ ] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR covers issue #< insert number here >.

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit 4648bae588e1c789c9ca1a15124a4dfe9467695a: Broken link result: [!] Error(s) found by the Links validator: 2 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:       19 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -25,18 +25,41 @@ After your PR has been submitted, we will review it as quickly as possible. This
...
-8. Create a Pull Request (PR) by going going to the [Create Pull Request page](https://github.com/OWASP/owasp-mstg/pull/new/master) and selecting your newly created branch. The target branch should typically be the Master branch. When submitting a PR, be sure to follow the checklist that is provided in the PR template. The checklist itself will be filled out by the reviewer.
-9. Your PR will be reviewed and comments may be given. In order to process a comment, simply make modifications to the same branch as before and push them to your repository. GitHub will automatically detect these changes and add them to your existing PR.
+4. Go the newly cloned directly "owasp-mstg" and add the remote upstream repository:
```
Comment (TheDauntless) [inline CONTRIBUTING.md]: Go *to* 

directly -> directory ?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -25,18 +25,41 @@ After your PR has been submitted, we will review it as quickly as possible. This
...
+origin git@github.com:<your Github handle>/owasp-mstg.git (push)
+$ git remote add upstream git@github.com:OWASP/owasp-mstg.git
+$ git remote -v
+origin git@github.com:<your Github handle>/owasp-mstg.git (fetch)
+origin git@github.com:<your Github handle>/owasp-mstg.git (push)
+upstream git@github.com:OWASP/owasp-mstg.git (fetch)
+upstream git@github.com:OWASP/owasp-mstg.git (push)
+```
+
+See also the GitHub documentation about [Configuring a remote for a fork](https://help.github.com/en/articles/configuring-a-remote-for-a-fork "Configuring a remote for a fork").
```
Comment (TheDauntless) [inline CONTRIBUTING.md]: about -> on

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -25,18 +25,41 @@ After your PR has been submitted, we will review it as quickly as possible. This
...
+8. After your modifications are done, push them to your forked repository. This can be done by executing the command `git add MYFILE` for every file you have modified, followed by `git commit -m 'Your Commit Message'` to commit the modifications and `git push` to push your modifications to GitHub.
+9. Create a Pull Request (PR) by going to your fork, <https://github.com/Your_Github_Handle/owasp-masvs> and click on the "New Pull Request" button. The target branch should typically be the Master branch. When submitting a PR, be sure to follow the checklist that is provided in the PR template. The checklist itself will be filled out by the reviewer.
```
Comment (TheDauntless) [inline CONTRIBUTING.md]: and click on  -> and clicking on

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -25,18 +25,41 @@ After your PR has been submitted, we will review it as quickly as possible. This
...
+10. Your PR will be reviewed and comments may be given. In order to process a comment, simply make modifications to the same branch as before and push them to your repository. GitHub will automatically detect these changes and add them to your existing PR.
+11. When doing a new PR in the future, make sure to always keep your local repo up to date:
```
Comment (TheDauntless) [inline CONTRIBUTING.md]: doing -> starting on

File: CONTRIBUTING.md (hunk 1 above)
Comment (sushi2k) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md (hunk 3 above)
Comment (sushi2k) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md (hunk 2 above)
Comment (sushi2k) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md (hunk 4 above)
Comment (sushi2k) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -25,7 +25,7 @@ After your PR has been submitted, we will review it as quickly as possible. This
 1. Create a GitHub account. Multiple different GitHub subscription plans are available, but you only need a free one. Follow [these steps](https://help.github.com/en/articles/signing-up-for-a-new-github-account "Signing up for a new GitHub account") to set up your account.
 2. Fork the repository. Creating a fork means creating a copy of the repository on your own account, which you can modify without any impact on this repository. GitHub has an [article that describes all the needed steps](https://help.github.com/en/articles/fork-a-repo "Fork a repo").
 3. Clone your own repository to your machine so that you can make modifications. If you followed the GitHub tutorial from step 2, you have already done this.
-4. Go the newly cloned directly "owasp-mstg" and add the remote upstream repository:
+4. Go the newly cloned directory "owasp-mstg" and add the remote upstream repository:
```
Comment (TheDauntless) [inline CONTRIBUTING.md]: Go the -> Go to the

**Comment by OMTGreleaser** [comment]
Results for commit 270f904f348a18525cb6338ccc1fece0ddab6757: Broken link result: [!] Error(s) found by the Links validator: 1 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:       16 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Pull request merged**